coprocessor/fhevm-engine/coprocessor/src/tfhe_worker.rs

```diff
@@ -117,21 +118,67 @@ async fn tfhe_worker_cycle(
         let now = std::time::SystemTime::now();
         let the_work = query!(
```

Comment:
Indentation in SQL is not helping, Is the indentation broken by cargo fmt ? (can be locally disabled)